app/javascript/shared/components/tags.jsx

```diff
@@ -439,7 +440,7 @@ class Tags extends Component {
           placeholder={`${maxTags} tags max, comma separated, no spaces or special characters`}
           autoComplete="off"
           value={defaultValue}
-          onInput={this.handleInput}
+          onInput={debounce(this.handleInput, 500)}
```

Comment:
Same thought as what I commented [here](https://github.com/thepracticaldev/dev.to/pull/6322/files#r384821491)

Reply:
Since we are only doing it in one place here I am fine to leave this the way it is. How does that sound?

Reply:
For consistency's sake, I'd say that it's worth it to pull this out into a function. I can just imagine someone doing a control + F for debounce and looking for how we've done it in other places, and then repeating the `onInput={debounce(...)}` pattern in the future, which is why I'd lean towards doing it the same way as #6103

Reply:
You got it!